v3/ldap.go

```diff
@@ -82,6 +83,13 @@ var BeheraPasswordPolicyErrorMap = map[int8]string{
 	BeheraPasswordInHistory:           "New password is in list of old passwords",
 }
 
+var logger = log.Default()
```

Comment:
so we can support Go v1.14 and v1.15, what do you think about this:
```suggestion
var logger = log.New(os.Stderr, "", log.LstdFlags)
```

Reply:
no worries; that works 👍